src/Offer/ValueObjects/BookingAvailabilityType.php

```diff
@@ -6,6 +6,10 @@
 
 use InvalidArgumentException;
 
+/**
+ * @deprecated
+ *   Use Use CultuurNet\UDB3\Model\ValueObject\Calendar\BookingAvailabilityType instead where possible.
```

Comment:
Small typo here 😉

Reply:
Thanks, I'll fix it! I've also found more VOs 🙈

Reply:
Fixed and also deprecated remaining VOs I found